Configuration/Services.yaml

```diff
@@ -22,3 +22,10 @@ services:
     tags:
       - name: 'console.command'
         command: 'productmanager:removeorphanattributevalues'
+
+  Pixelant\PxaProductManager\UserFunction\Solr\AbstractValue:
+    public: true
+  Pixelant\PxaProductManager\UserFunction\Solr\AttributeMultiValue:
+    public: true
+  Pixelant\PxaProductManager\UserFunction\Solr\AttributeValue:
+    public: true
```

Comment:
I haven't worked a lot with the new Symfony DI, but would it be an idea to set `_defaults.public = true` instead of defining it for each class?

Reply:
"`public: false` is a performance optimization and is therefore suggested to be set in extensions"

Reply:
Kult!